diskqueue.go

```diff
@@ -132,30 +136,30 @@ func NewWithDiskSpace(name string, dataPath string,
 	maxBytesDiskSpace int64, maxBytesPerFile int64,
 	minMsgSize int32, maxMsgSize int32,
 	syncEvery int64, syncTimeout time.Duration, logf AppLogFunc) Interface {
-	diskLimitFeatIsOn := true
+	enableDiskLimitation := true
 	if maxBytesDiskSpace <= 0 {
 		maxBytesDiskSpace = 0
```

Comment:
Could we delete this line `maxBytesDiskSpace = 0`?
If enableDiskLimitation is false, maxBytesDiskSpace is not used.

Reply:
Nice catch